src/M68k_LINE9.c

```diff
@@ -329,6 +359,21 @@ uint32_t *EMIT_SUBA_reg(uint32_t *ptr, uint16_t opcode, uint16_t **m68k_ptr)
     return ptr;
 }
 
+/****************************************************************************
+* 1001reg1mm00mxxx SUBX                                                     *
+*****************************************************************************
+* SUBX Dx,Dy|-(Ax),-(Ay)                                                    *
+*                                                                           *
+* Operation: dest - src → dest                                              *
```

Comment:
```suggestion
* Operation: dest - src - X → dest                                          *
```

Reply:
Yup! I was just hoping @CrypticalCode0 will change it ;)